anycloud/cli/src/deploy/mod.rs

```diff
@@ -1106,16 +1106,26 @@ pub async fn upgrade(
   let resp = post_v1("upgrade", body).await;
   let res = match resp {
     Ok(_) => {
+      // Check every 10s over 5 min if app already start upgrading
```

Comment:
is 5 mins too long to wait? The deploy service should be initializing the upgrade pretty quickly right?

Reply:
I mean, this would be the maximum amount of time we wait until we are sure we can start polling. Once we are sure, we `break`. Probably we could reduce it but to something considerable like 2 or 3 minutes. It is true that should be a quick action but it is not immediate, so I was having troubles since it was not polling for me since it checked, the size haven't change so it assumed it was ready and finish the `clap` interaction.

Reply:
gotcha - I had assumed that the network latency would be enough :/